data/developerhub/applications.json

```diff
@@ -181,6 +181,19 @@
       "complexity": ["intermediate"],
       "pro": true,
       "cloudPods": false
+    },
+    {
+      "title": "Search application with Lambda, Kinesis, Firehose, ElasticSearch, S3",
+      "description": "Fuzzy Search application demonstrates how to fuzzy-search a movie database using various AWS services, deployed using Terraform and AWS CLI on LocalStack",
+      "url": "https://github.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch",
+      "teaser": "https://raw.githubusercontent.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch/main/images/system_overview.png",
+      "services": ["s3", "lmb", "kns", "es"],
```

Comment:
firehose is missing from the services list, so we won't be able to filter it. 
or do we use the term "kinesis" for both  kinesis firehose + kinesis data stream?

Reply:
Let's use the term "kinesis" for both kinesis firehose + kinesis data stream (for now!) 👍